internal/contextkey.go

```diff
@@ -0,0 +1,29 @@
+// Package internal exists so that we can share the Context key functions between
+// mgohttp & mgohttptest without making them public.
```

Comment:
maybe include the reasons for why things are internal as comments

Reply:
Added in https://github.com/Clever/mgohttp/pull/2/commits/a956bb65ca8b0ad118ecdadceaf1422bb64e3043